[Bug]: Error while running command 'mypy src'

### Describe the bug

Found 2 errors after clone the master branch while running 'mypy src'. It seems the tuple list doesn't have the expected size. Here is the log:
src\fundus\parser\utility.py:505: error: List item 9 has incompatible type "tuple[str, str, str, str]"; expected "tuple[str, str] | tuple[str, str, str]"  [list-item]
src\fundus\parser\utility.py:507: error: List item 11 has incompatible type "tuple[str, str, str, str]"; expected "tuple[str, str] | tuple[str, str, str]"  [list-item]
Found 2 errors in 1 file (checked 208 source files)

### How to reproduce

```python
Run 'mypy src' in console
```

### Expected behavior.

Command executed successfully

### Logs and Stack traces

```stacktrace

```

### Screenshots

[fundus_mypy_src.txt](https://github.com/user-attachments/files/23058748/fundus_mypy_src.txt)

<img width="908" height="497" alt="Image" src="https://github.com/user-attachments/assets/2619a246-3c6c-4cf2-b0fb-90eeeaef45bb" />

### Additional Context

_No response_

### Environment

```markdown
Fundus branch Master

attrs==25.4.0
beautifulsoup4==4.14.2
black==23.1.0
Brotli==1.1.0
certifi==2025.10.5
chardet==5.2.0
charset-normalizer==3.4.4
click==8.3.0
colorama==0.4.6
cssselect==1.3.0
dict2xml==1.7.7
dill==0.4.0
FastWARC==0.15.2
feedparser==6.0.12
-e git+https://github.com/vrdhn91/fundus.git@f9c5062b83cb74ee922c4ab3efb649fde7b21846#egg=fundus
idna==3.11
iniconfig==2.3.0
isort==5.12.0
langdetect==1.0.9
lxml==5.4.0
more-itertools==9.1.0
mypy==1.9.0
mypy_extensions==1.1.0
packaging==25.0
pathspec==0.12.1
platformdirs==4.5.0
pluggy==1.6.0
pytest==7.2.2
python-dateutil==2.9.0.post0
requests==2.32.5
sgmllib3k==1.0.0
six==1.17.0
soupsieve==2.8
tqdm==4.67.1
types-colorama==0.4.15.20250801
types-html5lib==1.1.11.20251014
types-lxml==2025.8.25
types-python-dateutil==2.9.0.20251008
types-requests==2.32.4.20250913
typing_extensions==4.15.0
urllib3==2.5.0
validators==0.35.0
xmltodict==0.15.1
```

--- a/src/fundus/parser/utility.py
+++ b/src/fundus/parser/utility.py
@@ -506,6 +506,7 @@ class CustomParserInfo(parser.parserinfo):
         ("Nov", "November"),
         ("Dec", "December", "Dezember", "Dez"),
     ]  # type: ignore[assignment]
+    # type ignore due to types-python-dateutil==2.9.0.20251008, see https://github.com/flairNLP/fundus/issues/806
 
 
 def generic_date_parsing(date_str: Optional[str]) -> Optional[datetime]:

